willow/lib/tasks/reindex_test_data.rake

```diff
@@ -0,0 +1,88 @@
+namespace :willow do
+  desc "Reindex Willow test data (do not run on production!)"
```

Comment:
If we are quite sure and strict about this restriction, could we check RAILS_ENV and stop with an error if it's "production"?